airflow/providers/google/cloud/links/dataflow.py

```diff
@@ -48,5 +48,5 @@ def persist(
         operator_instance.xcom_push(
             context,
             key=DataflowJobLink.key,
-            value={"project_id": project_id, "location": region, "job_id": job_id},
+            value={"project_id": project_id, "region": region, "job_id": job_id},
```

Comment:
Same question

Reply:
Not improvement but fixing the link work, without that change link for Dataflow won't work cause of KeyError exception if I remember correctly